osu.Game/Overlays/BeatmapSet/PreviewButton.cs

```diff
@@ -80,7 +80,7 @@ protected override void Update()
         {
             base.Update();
 
-            if (Playing.Value && preview != null)
+            if (Playing.Value && preview != null && preview.Length > 0)

```

Comment:
what is this doing in this PR?

Reply:
Whoops, that was a makeshift fix for the Direct panels crashing (has already been resolved in another PR as far as I'm aware). Will remove in a second